app/actions/NoCacheActionBuilder.scala

```diff
@@ -0,0 +1,17 @@
+package actions
+
+import play.api.mvc._
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class NoCacheActionBuilder(
+  val parser: BodyParser[AnyContent],
+  val executionContext: ExecutionContext
+)
+    extends ActionBuilder[Request, AnyContent] {
+  implicit private val ec = executionContext
+
+  override def invokeBlock[A](request: Request[A], block: (Request[A]) => Future[Result]): Future[Result] =
+    block(request).map(_.withHeaders("Cache-Control" -> "no-cache, private", "Pragma" -> "no-cache"))
```

Comment:
Worth putting Expires: 0 here as well? https://stackoverflow.com/questions/49547/how-to-control-web-page-caching-across-all-browsers